src/lighthouse.c

```diff
@@ -705,7 +705,12 @@ int main(int argc, char **argv) {
   /* Set up the remote process. */
   int32_t to_child_fd, from_child_fd;
 
-  char *exec_file = settings.cmd;
+  char *exec_file;
+  if(access(settings.cmd, F_OK) == -1) {
+    exec_file = "/usr/share/lighthouse/cmd.py";
```

Comment:
this isn't particularly informative/portable.  perhaps mimic lines 716-718? something like this:

```c++
fprintf(stderr, "No 'cmd' set, please edit your lighthouserc or run lighthouse-install");
exit_code = 1;
return exit_code;
```